src/fshtypes/Configuration.ts

```diff
@@ -100,10 +100,10 @@ export type Configuration = {
   // refers to the IG.definiton.template definitions; not the publisher template in ig.ini.
   templates?: ImplementationGuideDefinitionTemplate[];
 
-  // DEPRECATED: Use of the template property is no longer recommended.  Authors should manage the
+  // NO LONGER SUPPORTED: Use of the template property is no longer recommended.  Authors should manage the
```

Comment:
Yes, I left this in the type because we still log errors based on `config.template` in IGExporter. I think if we ever remove those errors completely, it would be nice to clean up the type here to reflect what is actually supported.

Reply:
> And I think we should we similarly indicate that history is NOT SUPPORTED.

This was uncommitted on my computer from last week....

Reply:
Updated in f8706fe